### ISSUE TITLE
USB microphone

### ISSUE THREAD

**Issue by ghost** [bitplane/Microphone#1]
Would it also be possible to feed the audio through USB, thus acting as a USB microphone?'

**Comment by bitplane** [comment]
Yeah but it'd need to expose itself as a USB device, using the recent USB API (if that's possible). It'd be a completely different approach, far more involved than just "read here, write there"

**Comment by Kristinita** [comment]
+1
